bbb_view.php

```diff
@@ -188,16 +187,16 @@
 /**
  * Helper for getting the playback url that corresponds to an specific type.
  *
- * @param  string   $href
- * @param  string   $rid
- * @param  string   $rtype
+ * @param string $href
+ * @param string $rid
+ * @param string $rtype
  * @return string
  */
 function bigbluebuttonbn_bbb_view_playback_href($href, $rid, $rtype) {
     if ($href != '') {
         return $href;
     }
-    $recording = recording::read($rid);
+    $recording = recording::read_by(['recordingid' => $rid]);
```

Comment:
This is not correct. The value of rid must  correspond to mdl_bigbluebuttonbn_recordings.id because we are no longer exposing the actual recording id. The search must be done based on the sequential number Moodle generates.

Reply:
I stand to be corrected, but I changed that a bit last minute because the rid was in fact a recordingid (the string/bigbluebuttonid)